source/declarations.tex

```diff
@@ -227,12 +227,12 @@
 
 \pnum
 A \grammarterm{nodeclspec-function-declaration} shall declare a
-constructor, destructor, or conversion function.  \footnote{The
+constructor, destructor, or conversion function.\footnote{The
 ``implicit int'' rule of C is no longer supported.}
 \begin{note}
 A \grammarterm{nodeclspec-function-declaration} can only be used in a
 \grammarterm{template-declaration}~(Clause~\ref{temp}),
-\grammarterm{explicit-instantiation}~(\ref{temp.explicit}), or
+\grammarterm{explicit-instan\-tiation}~(\ref{temp.explicit}), or
```

Comment:
My more natural reaction would be to remove the non-breaking space first. I always thought that those were tenuous at best; there's very little reason to prevent a break before the reference. Either the reference fits on the same line and all is well, or it doesn't, and then that's still by far the best place to break compared to anywhere else.